vcpkg.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
is vcpkg adding anything to this PR? If its not being utilized/tested in the pipeline and it isnt explained in the README, would it be better to leave it out for now?

Reply:
Sure, I'll move it to a separate PR and add a CI pipeline for it.
I included it initially just in case you had not decided on which package manager you would like to incorporate into the repo, but that's no longer relevant.